### PR TITLE
Feature/custom state fields

### PR DESCRIPTION
**Pull request by abdullahcalisir12**
adding custom state field feature is added/

### CODE REVIEW

File: src/main/frontend/src/routes/_layout.svelte [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <script>
+<<<<<<< HEAD
```
Comment (hurelhuyag) [inline src/main/frontend/src/routes/_layout.svelte:2]: I same several this kind of lines. I think this shouldn't be here.

**Review by hurelhuyag — CHANGES_REQUESTED** [review]
I think those git merge errors needs to be fixed.

**Comment by VaughnVernon** [comment]
@abdullahcalisir12 We can't build to test. There are merge conflicts in the Svelte code. Neither @hurelhuyag nor I are able to review Svelte code but we were trying to do usability testing.

**Review by VaughnVernon — COMMENTED** [review]
@abdullahcalisir12 Would you also please support the basic type `Date`?

It is a placeholder type that will be translated for each platform, such as Java `LocalDate` and .NET `DateTime`.

**Comment by abdullahcalisir12** [comment]
Sorry @VaughnVernon @hurelhuyag for the merge conflicts, I wasn't aware of them. I just updated PR, can you review it?

**Review by hurelhuyag — APPROVED** [review]

**Comment by hurelhuyag** [comment]
I tried to run. Comboboxes, Modal dialogs not working.

**Comment by hurelhuyag** [comment]
I also noticed this weird animation. Is this related to PR? or different issue?
Screencast 2021-03-02 12:35:21

**Review by hurelhuyag — APPROVED** [review]

**Pull request merged**